src/table/cells/handleKeyPress.js

```diff
@@ -0,0 +1,74 @@
+const moveFocus = (tableRows, nextRow, nextCol) => {
+  const nextCell = tableRows[nextRow].getElementsByClassName('sn-table-cell')[nextCol];
+  nextCell.focus();
+  nextCell.setAttribute('tabIndex', '0');
+};
+
+const arrowKeysNavigation = (evt, rowAndColumn, rowIndex, colIndex) => {
+  let nextRow = rowIndex;
+  let nextCol = colIndex;
+
+  switch (evt.key) {
+    case 'ArrowDown':
+      if (rowIndex + 1 < rowAndColumn.tableRowSize) nextRow = ++nextRow;
+      break;
+    case 'ArrowUp':
+      if (rowIndex > 0) nextRow = --nextRow;
+      break;
+    case 'ArrowRight':
+      if (colIndex < rowAndColumn.tableColumnSize - 1) nextCol = ++nextCol;
+      break;
+    case 'ArrowLeft':
+      if (colIndex > 0) nextCol = --nextCol;
+      break;
+    default:
+  }
+
+  return {
+    tableRows: rowAndColumn.tableRows,
+    nextRow,
+    nextCol,
+  };
+};
+
+const tableRowAndColumn = (rootElement) => {
+  const tableRows = rootElement.getElementsByClassName('sn-table-row');
+  const tableRowSize = tableRows.length;
+
+  const headCells = rootElement.getElementsByClassName('sn-table-head-cell');
+  const tableColumnSize = headCells.length;
+
+  return { tableRows, tableRowSize, tableColumnSize };
+};
+
+const removeFocus = (evt) => {
+  evt.target.blur();
+  evt.target.setAttribute('tabIndex', '-1');
+};
+
+const preventDefaultBehavior = (evt) => {
+  evt.stopPropagation();
+  evt.preventDefault();
+};
+
+const handleKeyPress = (evt, rootElement, rowIndex, colIndex) => {
+  preventDefaultBehavior(evt);
+  switch (evt.key) {
+    // TODO page up/down etc
+    case 'ArrowUp':
+    case 'ArrowDown':
+    case 'ArrowRight':
+    case 'ArrowLeft': {
+      removeFocus(evt);
+      const rowAndColumn = tableRowAndColumn(rootElement);
```

Comment:
It would be really nice if these could be stored so we don't have to get them for every arrow key press. But I don't have a good idea of how to do it, so this is fine for now.

Reply:
All though I think it should be renamed to `getRows` since it is a getter